### PR TITLE
add hiplot

### PR DESCRIPTION
**Pull request by rpanai**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/hiplot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: ad984b72b98c6adbfba958d897ff42738882e1cd07b8f4f835f3f845fb147803
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - beautifulsoup4
```
Comment (npavlovikj) [inline recipes/hiplot/meta.yaml]: Usually `pip` and `python` are the only dependencies that should go in `host`, and everything else goes to `run`. Can you please try that?

File: recipes/hiplot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  imports:
+    - hiplot
+
+about:
+  home: "https://github.com/facebookresearch/hiplot"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "High dimensional Interactive Plotting tool"
+
```
Comment (npavlovikj) [inline recipes/hiplot/meta.yaml]: You can add `doc_url: https://facebookresearch.github.io/hiplot/`.

File: recipes/hiplot/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
```
Comment (npavlovikj) [inline recipes/hiplot/meta.yaml]: Please remove extra line.

Comment (rpanai) [reply recipes/hiplot/meta.yaml]: Done!

**Comment by npavlovikj** [comment]
The recipe LGTM @rpanai , thank you for contributing 🎉

**Pull request merged**